pythonforandroid/python.py

```diff
@@ -382,6 +390,32 @@ class HostPythonRecipe(Recipe):
     '''The default url to download our host python recipe. This url will
     change depending on the python version set in attribute :attr:`version`.'''
 
+    @property
+    def python_bin(self):
+        '''
+        Returns the name of the python executable depending on the build
+        platform since, as explained in python docs, some platforms generates a
+        different executable name to avoid conflicts with case insensitive File
+        systems (like Mac Os X).
+
+        .. seealso::
+            https://github.com/python/cpython/blob/3.7/README.rst#build-instructions
+
+        '''
+        if is_darwin():
+            return 'python.exe'
```

Comment:
I'd like to go back to my comments, this will break some Darwin installs.

This patch should not be based on darwin tests at all, darwin can has both insensitive or sensitive for both HFS+ and AFS, so in the case of a casesensitive darwin install, this will break.

A new test for FS case sensitivity should be make and used for both he patches and the binary name IMHO

is_darwin() should be is_caseinsensitivefs()